src/components/content-tab-assistant.tsx

```diff
@@ -26,23 +26,55 @@ const AuthenticatedView = memo(
 	( {
 		messages,
 		isAssistantThinking,
+		updateMessage,
+		path,
 	}: {
 		messages: MessageType[];
 		isAssistantThinking: boolean;
-	} ) => (
-		<>
-			{ messages.map( ( message, index ) => (
-				<ChatMessage key={ index } id={ `message-${ index }` } isUser={ message.role === 'user' }>
```

Comment:
It appears the `id` attribute was removed. The removal introduces a regression where screen readers are no longer able to navigate each message as a group, which was enabled in #248. 

I presume this was oversight and like a merge conflict resolution.

Reply:
Thanks for catching these and adding the PR to fix. Yeah that was a large conflict merge